_docs/reference/telemetry.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: guide
+title: StorageOS Telemetry
+anchor: reference
+module: reference/telemetry
+---
+
+# StorageOS Telemetry
+
+StorageOS collects telemetry and error reports  from StorageOS clusters via two
+different methods for two different purposes.
+
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to
+[sentry.io](https://sentry.io). This information helps our developers monitor
+and fix crashes.
+
+Information is sent to Sentry.io once per day or when a process inside the
```

Comment:
Inconsistent caps here - Exclusive of the heading, do we want Sentry.io or sentry.io?